docs/developer-guide/extensions/ui-extensions.md

```diff
@@ -130,26 +130,29 @@ It is also possible to add an optional flyout widget to your extension. It can b
 Below is an example of an extension using the flyout widget:
 
 ```typescript
```

Comment:
This is also an example, so my suggestion was to make this annotation JS and remove the type annotations on the props. I wouldn't change any other formatting or ordering and keep the diff light (right now you split out the code blocks etc, which I wouldn't recommend and actually feels confusing to me)